src/tsickle_compiler_host.ts

```diff
@@ -209,6 +214,15 @@ export class TsickleCompilerHost implements ts.CompilerHost {
     return tscSourceMapGenerator.toString();
   }
 
+  combineInlineSourceMaps(filePath: string, tscOutputText: string): string {
+    const sourceMapJson = extractInlineSourceMap(tscOutputText);
+    const composedSourceMap = this.combineSourceMaps(filePath, sourceMapJson);
+    const encodedSourceMap = encode(composedSourceMap);
+    return tscOutputText.replace(
+        new RegExp('//# sourceMappingURL=data:application/json;base64,(.*)'),
```

Comment:
Should these regexps be more specific?  For example, should they be anchored to the beginning of the line?  Does "." match newlines?